EmailValidator/Validation/NoRFCWarningsValidation.php

```diff
@@ -3,11 +3,39 @@
 namespace Egulias\EmailValidator\Validation;
 
 use Egulias\EmailValidator\EmailLexer;
+use Egulias\EmailValidator\Exception\InvalidEmail;
+use Egulias\EmailValidator\Validation\Error\RFCWarnings;
 
 class NoRFCWarningsValidation extends RFCValidation
 {
+    /**
+     * @var InvalidEmail
+     */
+    private $error;
+
+    /**
+     * {@inheritdoc}
+     */
     public function isValid($email, EmailLexer $emailLexer)
     {
-        return parent::isValid($email, $emailLexer) && empty($this->getWarnings());
+        if (!parent::isValid($email, $emailLexer)) {
+            return false;
```

Comment:
I've just realised that in this case we should probably return parent errors, right?